client/src/components/DataImportExport.vue

```diff
@@ -142,7 +144,10 @@ export default defineComponent({
         <v-card-title class="title">
           Import
         </v-card-title>
-        <v-card-text>
+        <v-card-text v-if="currentProject.id == 'global'">
```

Comment:
Given that we use this same condition in a multiple places, let's make `isGlobal` a getter in the Vuex store.

Reply:
changed in f932555